schemas/nsip-project.schema.json

```diff
@@ -4,39 +4,113 @@
 	"title": "NSIP Project",
 	"description": "Subset of Pins Data Model [Case]",
 	"type": "object",
-	"required": ["caseId", "sourceSystem"],
+	"required":[
+		"caseId",
+		"caseReference",
+		"projectName",
+		"projectDescription",
+		"decision",
+		"publishStatus",
+		"sector",
+		"projectType",
+		"sourceSystem",
+		"stage",
+		"projectLocation",
+		"projectEmailAddress",
+		"regions",
+		"transboundary",
+		"easting",
+		"northing",
+		"welshLanguage",
+		"mapZoomLevel",
+		"secretaryOfState",
+		"_preApplicationDates",
```

Comment:
These aren't required and perhaps should be removed. They were original intended as "comments". Planning to move to JSONC to support that properly.

Same for all the ones starting `_`.

Reply:
Resolved